app/assets/javascripts/jquery.infinite-pages.js.coffee

```diff
@@ -58,7 +58,7 @@ Released under the MIT License
     # load event if close enough
     check: ->
       nav = @$container.find(@options.navSelector)
-      if nav.size() == 0
+      if nav.length == 0
```

Comment:
Maybe we should use something like for older versions.
```
typeof nav.size == 'number' ? nav.size : nav.length 
```

Reply:
@zmillman Thanks! Check this [commit](https://github.com/magoosh/jquery-infinite-pages/pull/27/commits/807a3505afa1a07b03d2353d69af6d92c29dd97f).

Reply:
Cool!
👍